.github/scripts/deepseek_review.py

```
@@ -1,145 +1,69 @@
 """DeepSeek AI code review script called by deepseek-pr-review.yml.
 
-DeepSeek provides an OpenAI-compatible chat completions API at
-https://api.deepseek.com/v1/chat/completions. The integration reuses the
-shared `review_common` helpers so the prompt, verdict format, and error
-handling stay identical across all three reviewers.
+This module delegates entirely to ``cidaid_devtools.lib.deepseek_review``,
+passing an allotmint-specific ``RepoProfile`` so the prompt persona, stack
+description, known facts, and dimension‑3 safety checks match allotmint
```

Ruff (0.16.1)

[warning] Docstring contains ambiguous `‑` (NON-BREAKING HYPHEN). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)